Project-Lily/Project-Lily/DogListViewController.swift

```diff
@@ -12,11 +12,12 @@ class DogListViewController: UIViewController {
     
     //MARK: - Properties
     
-    var dogDatabase = DogDatabase()
+    //var dogDatabase = DogDatabase()
```

Comment:
Not sure if you'll need this later, but the good news is version control can help you out if you do! 😄 I'd recommend deleting this line